### DISCUSSION TITLE
Support for Oracle Object Store in Nextflow

### DISCUSSION THREAD

**Discussion by robnewman** [nextflow-io/nextflow#4571] [Ideas]
## New feature

Testing Nextflow running on an Oracle Kubernetes Engine (OKE) cluster and would like to access data stored in [Oracle Object Storage](https://www.oracle.com/cloud/storage/object-storage/) using the [Amazon S3 Compatibility API](https://docs.oracle.com/en-us/iaas/Content/Object/Tasks/s3compatibleapi.htm).

Does Nextflow support Oracle Object Store? If not, what is the plan to support Oracle Object Store?

**Comment by bentsherman** [comment]
If the Oracle storage is truly compatible with S3, you should be able to use it with some Nextflow configuration. See the [docs](https://nextflow.io/docs/latest/amazons3.html#s3-compatible-storage) for details.
